community-membership.md

```diff
@@ -13,7 +13,7 @@ delegated. They can be reached via e-mail cncf-opentelemetry-governance@lists.cn
 | **Role**   | **Responsibilities**                                  | **Requirements**                                             | **Defined by**                                               |
 | ---------- | ----------------------------------------------------- | ------------------------------------------------------------ | ------------------------------------------------------------ |
 | member     | active contributor in the community.  reviewer of PRs | sponsored by 2 approvers or maintainers. multiple contributions to the project. | OpenTelemetry GitHub org member.                             |
-| triager     | assist with project management and backlog organization. | nominated by a maintainer. attend meetings for one month. |  CONTRIBUTING, CODEOWNERS, or the README. |
+| triager     | assist with project management and backlog organization. | nominated by a maintainer. classify issues for one month. |  CONTRIBUTING, CODEOWNERS, or the README. |
```

Comment:
"classify issues" could be mistaken for "labeling issues" which would require triager rights

maybe something more generic like "help with issues for one month"?